Composer/packages/client/package.json

```diff
@@ -14,7 +14,8 @@
     "test": "jest",
     "lint": "eslint --quiet --ext .js,.jsx,.ts,.tsx ./src ./__tests__",
     "lint:fix": "yarn lint --fix",
-    "typecheck": "tsc --noEmit"
+    "typecheck": "tsc --noEmit",
+    "typecheck:watch": "tsc --noEmit --watch --noImplicitAny"
```

Comment:
I think this was left in unintentionally. I don't mind leaving it, but I would remove the `--noImplicitAny` because that differs from our tsconfig.

Reply:
It was intentional, but I'll remove the implicit-any thing.